### PR TITLE
docs: delete extra characters

### PR DESCRIPTION
**Pull request by liangliang4ward**
### Description
remove extra charactor

Fixes # (issue)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Review by tokers — DISMISSED** [review]

File: docs/zh/latest/plugins/limit-req.md [hunk 1]
```diff
@@ -30,12 +30,12 @@ title: limit-req
...
 | rate          | integer | 必须   |        | rate > 0                                                                | 指定的请求速率（以秒为单位），请求速率超过 `rate` 但没有超过 （`rate` + `burst`）的请求会被加上延时。                                             |
-| burst         | integer | 必须   |        | burst >= 0                                                              | t请求速率超过 （`rate` + `burst`）的请求会被直接拒绝。                                                                                            |
-| key           | string  | 必须   |        | ["remote_addr", "server_addr", "http_x_real_ip", "http_x_forwarded_for", "consumer_name"] | 用来做请求计数的依据，当前接受的 key 有："remote_addr"(客户端IP地址), "server_addr"(服务端 IP 地址), 请求头中的"X-Forwarded-For" 或 "X-Real-IP"，"consumer_name"(consumer 的 username)。 |
+| burst         | integer | 必须   |        | burst >= 0                                                              | 请求速率超过 （`rate` + `burst`）的请求会被直接拒绝。                                                                                            |
```
Comment (hf400159) [inline docs/zh/latest/plugins/limit-req.md:33]:
```suggestion
| burst         | integer | 必须   |        | burst >= 0                                                              | 请求速率超过（`rate` + `burst`）的请求会被直接拒绝。                                                                                            |
```

File: docs/zh/latest/plugins/limit-req.md [hunk 2]
```diff
@@ -30,12 +30,12 @@ title: limit-req
...
 | rejected_msg       | string | 可选                                |            | 非空                                          | 当请求超过阈值被拒绝时，返回的响应体。                                                                                                                                                                                                             |
-| nodelay       | boolean | 可选   | false  |                                                                         | 如果 nodelay 为 true， 请求速率超过 `rate` 但没有超过 （`rate` + `burst`）的请求不会加上延迟, 如果是 false，则会加上延迟。 |
-| allow_degradation              | boolean  | 可选                                | false       |                                                                     | 当限速插件功能临时不可用时是否允许请求继续。当值设置为 true 时则自动允许请求继续，默认值是 false。|
+| nodelay       | boolean | 可选   | false  |                                                                         | 如果 nodelay 为 true， 请求速率超过 `rate` 但没有超过 （`rate` + `burst`）的请求不会加上延迟, 如果是 false ，则会加上延迟。 |
```
Comment (hf400159) [inline docs/zh/latest/plugins/limit-req.md:37]:
```suggestion
| nodelay       | boolean | 可选   | false  |                                                                         | 如果 nodelay 为 true， 请求速率超过 `rate` 但没有超过（`rate` + `burst`）的请求不会加上延迟, 如果是 false ，则会加上延迟。 |
```

Comment (liangliang4ward) [reply docs/zh/latest/plugins/limit-req.md]: done

File: docs/zh/latest/plugins/limit-req.md (hunk 1 above)
Comment (liangliang4ward) [reply docs/zh/latest/plugins/limit-req.md]: done

**Review by juzhiyuan — COMMENTED** [review]
Hi, @liangliang4ward, we just merged a PR[1] about Chinese Copyrriting autocorrect linter, would you please merge the master branch here? 😄 

[1] https://github.com/apache/apisix/pull/6568

**Comment by liangliang4ward** [comment]
> Hi, @liangliang4ward, we just merged a PR[1] about Chinese Copyrriting autocorrect linter, would you please merge the master branch here? 😄
> 
> [1] #6568

done

File: docs/zh/latest/plugins/limit-req.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ title: limit-req
...
 | rate          | integer | 必须   |        | rate > 0                                                                | 指定的请求速率（以秒为单位），请求速率超过 `rate` 但没有超过（`rate` + `burst`）的请求会被加上延时。                                             |
-| burst         | integer | 必须   |        | burst >= 0                                                              | t 请求速率超过（`rate` + `burst`）的请求会被直接拒绝。                                                                                            |
+| burst         | integer | 必须   |        | burst >= 0                                                              | 请求速率超过（`rate` + `burst`）的请求会被直接拒绝。                                                                                            |
 | key           | string  | 必须   |        | ["remote_addr", "server_addr", "http_x_real_ip", "http_x_forwarded_for", "consumer_name"] | 用来做请求计数的依据，当前接受的 key 有："remote_addr"(客户端 IP 地址), "server_addr"(服务端 IP 地址), 请求头中的"X-Forwarded-For" 或 "X-Real-IP"，"consumer_name"(consumer 的 username)。 |
```
Comment (zaunist) [inline docs/zh/latest/plugins/limit-req.md:34]:
```suggestion
| key           | string  | 必须   |        | ["remote_addr", "server_addr", "http_x_real_ip", "http_x_forwarded_for", "consumer_name"] | 用来做请求计数的依据，当前接受的 key 有："remote_addr"(客户端 IP 地址), "server_addr"(服务端 IP 地址), 请求头中的 "X-Forwarded-For" 或 "X-Real-IP"，"consumer_name"(consumer 的 username)。 |
```

Comment (liangliang4ward) [reply docs/zh/latest/plugins/limit-req.md]: done

**Review by juzhiyuan — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Review by starsz — APPROVED** [review]

**Pull request merged**